src/components/SearchResults/Appeal/index.js

```diff
@@ -1,33 +1,50 @@
 import React from "react";
-import { Table } from "antd";
+import { Table, Button, Popconfirm } from "antd";
 import Details from "./Details";
 import { getStringAttr } from "@utils/Parser/strUtils";
 import { getPaginationObject } from "../utils";
 
-const columns = [
-  {
-    title: "Medical",
-    dataIndex: "services",
-    key: "medical",
-    render: (services) => getStringAttr(services.medical),
-  },
-  {
-    title: "Food & Essential",
-    dataIndex: "services",
-    key: "nonmedical",
-    render: (services) => getStringAttr(services.nonmedical),
-  },
-  {
-    title: "Region",
-    dataIndex: "region",
-  },
-  {
-    title: "Submitted At",
-    dataIndex: "createdAt",
-  },
-];
+function SearchResults({ result, pagination, onPageChange, onShowSizeChange, onResultClose }) {
+  const columns = [
+    {
+      title: "Medical",
+      dataIndex: "services",
+      key: "medical",
+      render: (services) => getStringAttr(services.medical),
+    },
+    {
+      title: "Food & Essential",
+      dataIndex: "services",
+      key: "nonmedical",
+      render: (services) => getStringAttr(services.nonmedical),
+    },
+    {
+      title: "Region",
+      dataIndex: "region",
+    },
+    {
+      title: "Submitted At",
+      dataIndex: "createdAt",
+    },
+
+    {
+      title: "Action",
+      dataIndex: "_id",
+      render: (id, row) => row.status == 'open' ?(
+        <Popconfirm
```

Comment:
move this element to /SearchResults/utils.js, since the code is same and you only need to pass it a confirm function on param.